### ISSUE TITLE
Connect stuck on "Loading"

### ISSUE THREAD

**Issue by grantwest** [trezor/connect#731]
When the Trezor Connect Popup opens it just gets stuck on "Loading" even when I have a Trezor connected and unlocked. Bridge shows the Trezor connected. I can use Suite with no issues.

Running `https://connect.trezor.io/8/popup.html`
macOS Big Sur 11.0.1
Firefox: 84.0
Trezor Bridge: 2.0.30
Trezor Model T Firmware: 2.3.4

Bridge shows Trezor is connected:
<img width="565" alt="image" src="https://user-images.githubusercontent.com/4594800/102740968-b500c480-4316-11eb-9890-8c191d19cea2.png">

Here is the log from bridge:
```
2.0.30
...
GET /status/
127.0.0.1 - - [20/Dec/2020:22:49:20 -0600] "GET /status/ HTTP/1.1" 200 5006
GET /status/
127.0.0.1 - - [20/Dec/2020:22:46:20 -0600] "GET /status/ HTTP/1.1" 200 4652
GET /status/
127.0.0.1 - - [20/Dec/2020:18:38:34 -0600] "GET /status/ HTTP/1.1" 200 4812
GET /status/
```

I'm not sure how to debug further, but if you have a suggestion I would be happy to try it. The firefox web console has no errors or output in it.

**Comment by grantwest** [comment]
I cleared the cache as suggested by a Trezor support page but it made no difference. I created a new Firefox profile with all default settings and it worked. Must be an extension or privacy setting that is breaking Trezor Connect. As this is a frustrating experience for new users, I am willing to debug this further, but I need some guidance on next steps.

**Comment by szymonlesisz** [comment]
hi @grantwest, 
extensions (ad blockers) should be covered. you should see an information about blocked communication, like this:
![screenshot](https://user-images.githubusercontent.com/3435913/102754897-f2447100-436d-11eb-9522-2cbf830f8373.png)

however there might be some recent changes either in extensions (do you have any?) or firefox it self. idk i will try it on similar setup.

If you would like to debug it further see console network tab (in the source/implementation page, not in trezor-connect popup, popup is only a dummy UI and it doesn't do any logic so it should not display any errors) 

you can also enable additional log-level to the console (again, in the source page) by setting additional GET param:

`http://xxxyyyzzz.com/?trezor-connect-src=https://connect.trezor.io/8/`

or

`http://xxxyyyzzz.com/?trezor-connect-src=https://localhost:8888/` if you are running your local version of connect

**Comment by grantwest** [comment]
I did some testing and found exactly what was breaking it in my case. 
This is the setting I normally use:
<img width="914" alt="image" src="https://user-images.githubusercontent.com/4594800/102795350-4e10f900-4372-11eb-82e1-5ee358c2767f.png">

And this is what I changed it to to make it work:
<img width="918" alt="image" src="https://user-images.githubusercontent.com/4594800/102795409-63862300-4372-11eb-8108-730e4222f8b9.png">

The console shows this error:
```
Request to access cookie or storage on “https://connect.trezor.io/8/iframe.html?version=8.1.15&manifest=<censored>” was blocked because we are blocking all third-party storage access requests and content blocking is enabled.

Request to access cookie or storage on “https://connect.trezor.io/8/iframe.html?version=8.1.15&manifest=<censored>” was blocked because we are blocking all third-party storage access requests and content blocking is enabled.
```

I'm not sure if this is a fixable issue or not. Is accessing the cookie or storage from another site 100% required for it to function?

**Comment by szymonlesisz** [comment]
this should be fixed in 8.1.21, thanks for detailed report!
